src/components/forms/tabs/PrivatePerson.js

```diff
@@ -10,11 +12,16 @@ type Props = {
   prefix: string
 };
 
+const StyledContainer = styled(Container)`
+  margin-top: 2rem;
+  margin-bottom: 2rem;
```

Comment:
Could make it as common styled component that we can import/export and use. Makes changes easier and don’t need to duplicate rules. Ex. `/src/components/common/ContainerWithMargins.js`